core/tests/protractor_utils/ProfilePage.js

```diff
@@ -73,6 +75,32 @@ var ProfilePage = function() {
   this.expectUserToNotHaveInterestPlaceholder = function() {
     expect(interestPlaceholder.isPresent()).toBe(false);
   };
+
+  this.expectToHaveExplorationCards = function() {
+    allExplorationCardElements.then(function(cards) {
+      if (cards.length === 0) {
+        throw 'There is no exploration card on this profile';
+      }
+      expect(cards.length).toBeGreaterThan(0);
+    });
+  };
+
+  this.expectToHaveExplorationCardByName = function(explorationName) {
+    var explorationsCardByName = allExplorationCardElements.filter(
+      function(card) {
+        var cardTitle = card.element(
+          by.css('.protractor-test-exp-summary-tile-title'));
+        return cardTitle.
+          getText().then(function(title) {
+            return (title === explorationName);
+          });
```

Comment:
```
        return cardTitle.getText().then(function(title) {
            return title === explorationName;
          });
```

(Just to utilize the 80 space and break the line before `.`)

Reply:
Done